test/suites/remote.sh

```diff
@@ -22,6 +19,14 @@ test_remote_url() {
     lxc_remote remote add test "${url}"
     lxc_remote remote remove test
   done
+
+  # Check that we can add simplestream remotes with valid certs without confirmation
+  if [ -z "${LXD_OFFLINE:-}" ]; then
+    lxc_remote remote add ubuntu1 https://cloud-images.ubuntu.com/buildd/releases --protocol=simplestreams
+    lxc_remote remote add ubuntu2 https://cloud-images.ubuntu.com:443/buildd/releases --protocol=simplestreams
+    lxc_remote remote remove ubuntu1
+    lxc_remote remote remove ubuntu2
+  fi
```

Comment:
Based on #12414, those don't do much. Should I go ahead and add something like:

```
lxc image list ubuntu1: | grep -cwF CONTAINER
lxc image list ubuntu2: | grep -cwF CONTAINER
```

Reply:
sounds good.